src/condor_io/reli_sock.cpp

```diff
@@ -870,6 +936,91 @@ int ReliSock::RcvMsg::rcv_packet( char const *peer_description, SOCKET _sock, in
 		}
 	}
 
+		// Note that we don't check the header here as the body may be
+		// split across several function invocations.
+	if (!p_sock->get_encryption() && !p_sock->m_finished_recv_header && p_sock->m_recv_md_ctx && (p_sock->_bytes_recvd < 1024*1024)) {
+	/*	if (1 != EVP_DigestUpdate(p_sock->m_recv_md_ctx.get(), m_tmp->get_ptr(), m_tmp->num_untouched())) {
+			dprintf(D_NETWORK, "IO: Failed to update the message digest.\n");
+			return false;
+	        }
+		else {
+			dprintf(D_NETWORK, "AESGCM: Recv body digest added %u bytes \n", m_tmp->num_untouched());
+		}
+	*/}
```

Comment:
We want this to be re-enabled before final merge, right?

Reply:
I don't think I commented this out. I'm pretty sure that's how it was in the original PR so I figured you had left it out for a reason.  (num_untouched() make me wonder if these are bytes that hadn't yet been "processed" and were therefor not meant to be in the digest?  I would have to read the code more closely)

Reply:
BrianB made a commit that re-added this on send and receive side.